pkg/model/trafficpatterns/sinusoidal.go

```diff
@@ -61,7 +61,7 @@ func (s *sinusoidal) Generate() {
 	}
 }
 
-func NewSinusoidal(env simulator.Environment, source model.TrafficSource, buffer model.RequestsBufferedStock, config SinusoidalConfig) Pattern {
+func NewSinusoidal(env simulator.Environment, source model.TrafficSource, buffer model.RequestsRoutingStock, config SinusoidalConfig) Pattern {
```

Comment:
Rename `buffer`.

Reply:
Renamed to routingStock in https://github.com/pivotal/skenario/pull/67/commits/dfd4d60851eea3b87cc9d7e4dc950b95bc92c495 commit.